src/ft2font_wrapper.cpp

```diff
@@ -1567,6 +1610,9 @@ PYBIND11_MODULE(ft2font, m, py::mod_gil_not_used())
                 PyFT2Font_draw_glyph_to_bitmap__doc__);
         }
         cls
+        .def("_draw_glyph_at", &PyFT2Font__draw_glyph_at,
+             "image"_a, "x"_a, "y"_a, "glyph"_a, py::kw_only(), "antialiased"_a=true,
+             PyFT2Font__draw_glyph_at__doc__)
```

Comment:
How does this correlate with #30059?

Reply:
This is indeed redundant with the _render_glyph method of #30059; when both PRs are in I'll rewrite this in terms of the #30059 API.
However #30059 seems a bit further away from merging, both because of the API design choices at https://github.com/matplotlib/matplotlib/pull/30059#discussion_r2134176872, and because of the maybe-off-by-1 positioning issues mentioned at https://github.com/matplotlib/matplotlib/pull/30059#issuecomment-2961414822 (actually I haven't revisited that at all since then, so that may need a bit of re-checking).